libs/sof/sof_multicamera_gpu.cpp

```
@@ -66,60 +68,112 @@ void MultiSOFGPU::LaunchTrackingPrimaryToSecondary(CameraId primary_id, CameraId
   const ImageContextPtr secondary_image = curr_images[secondary_id];
 
   const camera::ICameraModel& intrinsicsP = *rig_.intrinsics[primary_id];
-  const camera::ICameraModel& intrinsicsS = *rig_.intrinsics[secondary_id];
 
-  PrimaryToSecondaryGPUTracker& tracker = secondary_from_primary_sof_[primary_id][secondary_id];
+  PrimaryToSecondaryGPUTracker& pair = secondary_from_primary_sof_.at(primary_id).at(secondary_id);
+  assert(pair.tracker != nullptr);
```

Clang (14.0.6)

[note] +1, including nesting penalty of 0, nesting level increased to 1

(clang)